Sprint-1/interpret/time-format-day-option.js

```diff
@@ -0,0 +1,11 @@
+const movieLength = 8784; // length of movie in seconds
+
+const remainingSeconds = movieLength % 60; // calculate remaining seconds
+const totalMinutes = (movieLength - remainingSeconds) / 60; // calculate total minutes
+const remainingMinutes = totalMinutes % 60; // calculate remaining minutes
+const totalHours = (totalMinutes - remainingMinutes) / 60; // calculate total hours
+const totalDays = Math.floor(totalHours / 24); // calculate total days
+const remainingHours = totalHours % 24; // calculate remaining hours
+
+const result = `${totalDays} days, ${remainingHours} hours, ${remainingMinutes} minutes, ${remainingSeconds} seconds`;
```

Comment:
Instead of using "result", try always to use a name variable that better describes the purpose of it.

Reply:
Thanks for the suggestion😊 I agree that using more descriptive variable names improves code readability. 

I'll rename result to something more meaningful like formattedTime to reflect its purpose better.